marshmallow/fields.py

```diff
@@ -1172,13 +1173,13 @@ def _deserialize(self, value, attr, data, **kwargs):
             self.fail('invalid')
 
 
-class Dict(Field):
-    """A dict field. Supports dicts and dict-like objects. Optionally composed
+class Mapping(Field):
+    """An abstract class for objects with key-value pairs. Optionally composed
     with another `Field` class or instance.
 
     Example: ::
 
-        numbers = fields.Dict(keys=fields.Str(), values=fields.Float())
+        numbers = fields.Mapping(keys=fields.Str(), values=fields.Float())
```

Comment:
We should probably drop this example, since `Mapping` isn't meant to be instantiated.

Reply:
Right. I missed to correct this.